spec/support/shared_examples/authenticated_action.rb

```diff
@@ -6,7 +6,7 @@
     perform_action
   end
 
-  it 'returns a 302 status code' do
-    expect(response.status).to eq(302)
+  it 'returns a 401 status code' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.